package.json

```diff
@@ -254,6 +254,11 @@
                             "markdownDescription": "%configuration.java.test.config.javaExec.description%",
                             "default": ""
                         },
+                        "encoding": {
```

Comment:
Since `java.test.config` can either be an array or an object. So we need to duplicated this under `item` section as well. (start from line 349)

Reply:
Also noted, because they're right next to each other, that it's `vmargs` in the array and `vmArgs` in the singular ; have changed the case of the array to match (this has been deprecated 4 years, I think it might have been long enough :-) )

Reply:
Thank you